pkg/graphql/errors.go

```diff
@@ -1,27 +1,73 @@
 package graphql
 
 import (
+	"fmt"
 	"io"
 )
 
+type Errors interface {
+	error
+	WriteResponse(writer io.Writer) (n int, err error)
+	Count() int
+}
+
 type OperationValidationErrors []OperationValidationError
 
-func (v OperationValidationErrors) AsGraphQLErrors(writer io.Writer) (n int, err error) {
-	return writer.Write(nil)
+func (o OperationValidationErrors) Error() string {
```

Comment:
Instead of using `WriteResponse`, we should use this function to show errors because in tyk middleware, returned error is also written to response. If you make it nil, it means the operation successful.

Reply:
I found a workaround to handle this situation in gateway, ignore this one.